backend/src/main/java/ai/verta/modeldb/project/ProjectDAORdbImpl.java

```diff
@@ -732,6 +732,8 @@ public Project deepCopyProjectForUser(String srcProjectID, UserInfo newOwner)
       LOGGER.debug("Mark Projects as deleted : {}, count : {}", allowedProjectIds, updatedCount);
       transaction.commit();
       LOGGER.debug("Project deleted successfully");
+
+      roleService.deleteProjectResources(allowedProjectIds);
```

Comment:
This must be done in the cronjob instead of the API call itself as it might fail. Since we already do it there, we should remove from here or we can end up sending an error for something that was as success. We should only do things once =)

Reply:
it is fine to remove this line from here.

Reply:
Fixed.